lib/src/Pos/Util/Servant.hs

```diff
@@ -280,6 +282,25 @@ instance HasSwagger (apiType a :> res) =>
     HasSwagger (WithDefaultApiArg apiType a :> res) where
     toSwagger _ = toSwagger (Proxy @(apiType a :> res))
 
+-------------------------------------------------------------------------
+-- HasClient instances we need for benchmarking.
```

Comment:
Just a random thought: if they are needed only for wallet benchmarking, maybe it's better to move them to benchmarks or at least to `wallet-new` library? This way `cardano-sl` library will have less dependencies.
Not sure what's the right place for these instances, to be honest :thinking:

Reply:
Yes, these instances are for benchmarking only. But we (@Martoon-00 and me) thought that if `HasServer` is here, so `HasClient` can be here as well... :-)